src/Handler/Info/ObjectInfoHandler.php

```diff
@@ -154,26 +154,16 @@ private function getObjectsForDataset(BigQueryClient $client, Dataset $dataset):
                 $table->id()
             ));
             $info = $table->info();
-            try {
-                // we need to run select as query as ->rows method is not supported for view
-                $client->runQuery($client->query(sprintf(
-                    'SELECT * FROM %s.%s.%s LIMIT 1',
-                    BigqueryQuote::quoteSingleIdentifier($info['tableReference']['projectId']),
-                    BigqueryQuote::quoteSingleIdentifier($info['tableReference']['datasetId']),
-                    BigqueryQuote::quoteSingleIdentifier($info['tableReference']['tableId']),
-                )));
```

Comment:
Toto byla kravina, má to dvě chyby:
1. půšťá sa to pod projektovým uživatelem a ten može mět práva číst externí tabulku
2. pokud má tabulka requirePartitionFilter tak to spadne